diff --git a/src/client/state/settings.js b/src/client/state/settings.js
index 32f55fc..358c94b 100644
--- a/src/client/state/settings.js
+++ b/src/client/state/settings.js
@@ -59,9 +59,17 @@ class Settings extends EventEmitter {
   applyTheme() {
     this._clearTheme();
     if (this.useSystemTheme) {
-      document.body.classList.add('system-theme');
+      // document.body.classList.add('system-theme');
+      QML.setTheme('System')
+      if (QML.settings.systemTheme === 'SuruDark')
+      {
+        document.body.classList.add('dark-theme')
+      }
     } else if (this.themes[this.themeIndex]) {
+      QML.setTheme(this.themes[this.themeIndex] === 'silver-theme' ? 'Ambiance' : 'SuruDark')
       document.body.classList.add(this.themes[this.themeIndex]);
+    } else {
+      QML.setTheme('Ambiance')
     }
   }
 
