From 920ca66972190687262e5539b0a841d9b8bda6e3 Mon Sep 17 00:00:00 2001
From: Marcel Alexandru Nitan <nitan.marcel@protonmail.com>
Date: Mon, 4 Apr 2022 21:19:30 +0300
Subject: [PATCH] Set push notifications to UTs push hook

---
 src/client/initMatrix.js | 35 +++++++++++++++++++++++++++++++++++
 1 file changed, 35 insertions(+)

diff --git a/src/client/initMatrix.js b/src/client/initMatrix.js
index f6fc9eb..663a2bf 100644
--- a/src/client/initMatrix.js
+++ b/src/client/initMatrix.js
@@ -17,6 +17,18 @@ class InitMatrix extends EventEmitter {
     await this.startClient();
     this.setupSync();
     this.listenEvents();
+
+    try {
+      this.setPush(QML.settings.pushToken, QML.settings.pushAppId)
+
+      QML.matrixPushTokenChanged.connect(async function (token) {
+        this.setPush(token, QML.settings.pushAppId)
+
+      });
+    }
+    catch {
+      console.log("Failed to update push rules!")
+    }
   }
 
   async startClient() {
@@ -46,6 +58,29 @@ class InitMatrix extends EventEmitter {
     this.matrixClient.setGlobalErrorOnUnknownDevices(false);
   }
 
+  setPush(token, appId) {
+    var promises = []
+
+    promises.push(
+    this.matrixClient.setPusher(
+      {
+        "app_display_name": cons.DEVICE_DISPLAY_NAME,
+        "app_id": appId,
+        "append": true,
+        "data": {
+            "url": "https://push.ubports.com:5003/_matrix/push/v1/notify"
+        },
+        "device_display_name": cons.DEVICE_DISPLAY_NAME,
+        "lang": "en",
+        "kind": "http",
+        "profile_tag": "xxyyzz",
+        "pushkey": token
+      },
+      function () {}
+    ))
+    this.matrixClient.setPushRuleEnabled('global', 'content', '.m.rule.contains_user_name', true, function () {})
+  }
+
   setupSync() {
     const sync = {
       NULL: () => {
-- 
2.30.2

