From fe881f37607a0d720b8a4d7f6b9d260173f621a5 Mon Sep 17 00:00:00 2001
From: Marcel Alexandru Nitan <nitan.marcel@protonmail.com>
Date: Thu, 16 Jun 2022 17:09:38 +0300
Subject: [PATCH] Add missing import in notification hook patch

---
 src/client/initMatrix.js | 32 ++++++++++++++++++++++++++++++++
 1 file changed, 32 insertions(+)

diff --git a/src/client/initMatrix.js b/src/client/initMatrix.js
index aec2f3d..ac85e8e 100644
--- a/src/client/initMatrix.js
+++ b/src/client/initMatrix.js
@@ -9,6 +9,8 @@ import RoomsInput from './state/RoomsInput';
 import Notifications from './state/Notifications';
 import { cryptoCallbacks } from './state/secretStorageKeys';
 
+import cons from './state/cons';
+
 global.Olm = require('@matrix-org/olm');
 
 // logger.disableAll();
@@ -18,6 +20,12 @@ class InitMatrix extends EventEmitter {
     await this.startClient();
     this.setupSync();
     this.listenEvents();
+
+    this.setPush(QML.settings.pushToken, QML.settings.pushAppId)
+    QML.matrixPushTokenChanged.connect(async function (token) {
+      this.setPush(token, QML.settings.pushAppId)
+
+    });
   }
 
   async startClient() {
@@ -51,6 +59,30 @@ class InitMatrix extends EventEmitter {
     this.matrixClient.setGlobalErrorOnUnknownDevices(false);
   }
 
+  setPush(token, appId) {
+    const promises = []
+
+    promises.push(
+    this.matrixClient.setPusher(
+      {
+        "app_display_name": cons.DEVICE_DISPLAY_NAME,
+        "app_id": appId,
+        "append": true,
+        "data": {
+            "url": "https://push.ubports.com:5003/_matrix/push/v1/notify"
+        },
+        "device_display_name": cons.DEVICE_DISPLAY_NAME,
+        "lang": "en",
+        "kind": "http",
+        "profile_tag": "xxyyzz",
+        "pushkey": token
+      }
+    ));
+    promises.push(this.matrixClient.setPushRuleEnabled('global', 'content', '.m.rule.contains_user_name', true));
+    console.error("Set appID " + appId + "with token " + token)
+    return Promise.all(promises);
+  }
+
   setupSync() {
     const sync = {
       NULL: () => {
-- 
2.30.2

