From 6f98e6bba13fb7bb345fd1713ffb684756b54ecf Mon Sep 17 00:00:00 2001
From: Marcel Alexandru Nitan <nitan.marcel@protonmail.com>
Date: Mon, 4 Apr 2022 21:29:35 +0300
Subject: [PATCH] Add credits

---
 src/app/organisms/settings/Settings.jsx | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/app/organisms/settings/Settings.jsx b/src/app/organisms/settings/Settings.jsx
index acfef5c..c55032a 100644
--- a/src/app/organisms/settings/Settings.jsx
+++ b/src/app/organisms/settings/Settings.jsx
@@ -240,6 +240,10 @@ function AboutSection() {
               {/* eslint-disable-next-line react/jsx-one-expression-per-line */ }
               <Text>The <a href="https://material.io/design/sound/sound-resources.html" target="_blank" rel="noreferrer noopener">Material sound resources</a> are © <a href="https://google.com" target="_blank" rel="noreferrer noopener">Google</a> used under the terms of <a href="https://creativecommons.org/licenses/by/4.0/" target="_blank" rel="noreferrer noopener">CC-BY 4.0</a>.</Text>
             </li>
+            <li>
+              {/* eslint-disable-next-line react/jsx-one-expression-per-line */ }
+              <Text>The <a href="https://github.com/nitanmarcel/cinny-click-packaging" target="_blank" rel="noreferrer noopener">cinny-click-packaging</a> for Ubuntu Touch is © <a href="https://github.com/nitanmarcel" target="_blank" rel="noreferrer noopener">Marcel Alexandru Nitan</a> used under the terms of <a href="https://www.gnu.org/licenses/gpl-3.0.en.html" target="_blank" rel="noreferrer noopener">GNU GENERAL PUBLIC LICENSE v3.0</a>.</Text>
+            </li>
           </ul>
         </div>
       </div>
-- 
2.30.2

