diff --git a/src/app/organisms/settings/Settings.jsx b/src/app/organisms/settings/Settings.jsx
index b50c992..cfad13b 100644
--- a/src/app/organisms/settings/Settings.jsx
+++ b/src/app/organisms/settings/Settings.jsx
@@ -270,11 +270,6 @@ const tabItems = [{
   iconSrc: SunIC,
   disabled: false,
   render: () => <AppearanceSection />,
-}, {
-  text: tabText.NOTIFICATIONS,
-  iconSrc: BellIC,
-  disabled: false,
-  render: () => <NotificationsSection />,
 }, {
   text: tabText.EMOJI,
   iconSrc: EmojiIC,
